index.html

```diff
@@ -21,150 +20,188 @@
-                    <li> <a href="www.angelink.com" target="_blank" rel="noopener"> <img src="img/angela.png" alt="angela"></a></li>
-                    <li> <a href="www.medium.com" target="_blank" rel="noopener"> <img src="img/M.png" alt="m"></a></li>
-                </ul>
-            </div>
+    <section class="header">
+        <h1 class="headline">I'm Daniel<br />Glad you're here!</h1>
+        <p class="headline-description">I'm a software developer. I can help you build a product, feature or website.
+            Look through some of my work and experience! If you like what you see and have a project you need coded,
+            donn't hesitate to contact me.</p>
+        <div>
+            <p class="connect">Let's connect</p>
+            <ul class="social">
+                <li> <a href="https://www.linkedin.com/in/daniel-njoroge-3651a2a5/" target="_blank" rel="noopener"><img
+                            src="img/Linkedin.png" alt="linkedin"></a> </li>
+                <li> <a href="https://twitter.com/Icharian" target="_blank" rel="noopener"> <img src="img/Twitter.png"
+                            alt="twitter"></a></li>
+                <li> <a href="https://github.com/Dinicharia" target="_blank" rel="noopener"> <img src="img/github.png"
+                            alt="github"></a></li>
+                <li> <a href="www.angelink.com" target="_blank" rel="noopener"> <img src="img/angela.png"
+                            alt="angela"></a></li>
+                <li> <a href="www.medium.com" target="_blank" rel="noopener"> <img src="img/M.png" alt="m"></a></li>
+            </ul>
         </div>
     </section>
-    <div class="gridsection">
-        <section class="cardworks">
-            <div class="container">
-                <div class="workimage"><img src="img/Snapshoot.png" alt="snapshoot"></div>
-                <h1 class="greet">Tonic</h1>
-                <div class="frame2">
-                    <div class="client">CANOPY</div>
-                    <div><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
-                    <div class="role">Back End Dev</div>
-                    <div><a href="#"> <img src="img/Counter.png" alt="snapshoot"></a></div>
-                    <div class="role">2015</div>
+    <section class="works">
+        <div class="work-container">
+            <div class="work">
+                <div class="img-container">
+                    <img src="img/Snapshoot.png" class="work-image" alt="work1">
                 </div>
-                <p class="para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
-                <div class="tags">
-                    <p class="tag">html</p>
-                    <p class="tag">css</p>
-                    <p class="tag">javascript</p>
+                <div class="work-details">
```

Comment:
- [ ] Kindly consider styling the project card as in the Figma design, with the same spacing (margin/padding), and every second card should be aligned in reverse. this will show your ability to stick to the provided design

|Your design|Figma design|
|-|-|
|![Screenshot_20220825_181208](https://user-images.githubusercontent.com/25227022/186728511-4c3d3932-43f1-4f78-b0c9-7ce35fc89c73.png)|![Screenshot_20220825_181646](https://user-images.githubusercontent.com/25227022/186728830-df279e7c-d07c-4ae8-aed2-daf4ebc883eb.png)|